astropy/io/fits/hdu/compressed.py

```diff
@@ -30,13 +31,7 @@
 from .image import ImageHDU
 from .table import BinTableHDU
 
-try:
-    from astropy.io.fits import compression
-
-    COMPRESSION_SUPPORTED = COMPRESSION_ENABLED = True
-except ImportError:
-    COMPRESSION_SUPPORTED = COMPRESSION_ENABLED = False
-
+COMPRESSION_ENABLED = True
```

Comment:
Because HDUList does some horrible things using this:

https://github.com/astropy/astropy/blob/1bac6191b70ae37f61e4dd83ed6f9ac325022aaa/astropy/io/fits/hdu/hdulist.py#L1302

Basically if you open the FITS file with ``disable_image_compression`` it will temporarily modify the global variable ``COMPRESSION_ENABLED``. We should definitely refactor this, but for now we were trying to not refactor too much.

Reply:
I've now added a comment above where COMPRESSION_ENABLED is defined to explain this.